BREAKING.md

```diff
@@ -83,6 +84,12 @@ The following functions, interfaces, and types currently available in `@fluidfra
 -   `ISummarizerNodeRootContract`
 -   `RefreshSummaryResult`
 
+### IFluidTokenProvider deprecated
+
+The IFluidTokenProvider interface has been deprecated and will be removed in an upcoming release. Fluid Framework does not prescribe a particular approach to token providers.
```

Comment:
I know for routerlicious/frs we rely on ITokenProvider and mostly on service/driver layer. But given that ILoaderOptions seem to be a generic property bag of options, it would be worthwhile checking there are no (existing) expectations from 1Ps here.

I love all the efforts you have put in to clean up some of the APIs here @ChumpChief

Reply:
This one is actually a different token provider concept (I think you had the same reaction @tylerbutler did :-D)

This provider was added to support an example (shared-text), and the token this refers to was for using some external intelligence/augloop service in conjunction with the document text.  Not related to the Azure service tokens.

Reply:
Yes, my comment confused the two, but ITokenProvider is a separate concept here. I think I’m clear :) I was asking about ILoaderOptions that the implementation of this token provider was surfacing. just wanted to make sure it’s not used by ffx/1ps.

Reply:
I could have been wrong here. This is only interface deprecation.